components/elm/src/main/elm_varctl.F90

```diff
@@ -446,8 +449,8 @@ module elm_varctl
 
   !---------------------------------------------------------------------------
   subroutine elm_varctl_set( caseid_in, ctitle_in, brnch_retain_casename_in,    &
-       single_column_in, scmlat_in, scmlon_in, nsrest_in, &
-       version_in, hostname_in, username_in)
+       single_column_in, scm_multcols_in, scmlat_in, scmlon_in, scm_nx_in, scm_ny_in, &
+       nsrest_in, version_in, hostname_in, username_in)
```

Comment:
`elm_varctl_set` is also called in `lnd_comp_esmf`. It doesn't seem that these newly added parameters are specified when it's called there. Is it unneeded in that case?